demos/wstrust/service/src/main/webapp/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,3 @@
+Manifest-Version: 1.0
+Dependencies: org.jboss.ws.cxf.jbossws-cxf-client
```

Comment:
We're adding these dependencies to the deployments' MANIFEST.MF files, can qe add a link to the documentation explaining the reason for it?

Reply:
I don't find link-able documentation.  It is required for code references to be resolved.

Reply:
Do we have additional resources that inspired this example? We might have a look around starting from there or even just link the original example and be done with it. I think it's important we add a rationale for specific configuration details. WDYT?

Reply:
done

Reply:
Thanks!